### PR TITLE
Review

### PR DESCRIPTION
**Pull request by jrybojad**
Review routing and form verification done.

### CODE REVIEW

File: webApp/server_modules/routes/profile.js [hunk 1]
```diff
@@ -36,27 +36,62 @@ router.get('/user/:id(\\d+)', (req, res) => {
...
             }));
             res.render('profileView-guest', { profile: prof, courses });
-        }).catch(() => {
-            throw new Error(`No such profile ${googleID}`);
+        }).catch((err) => {
+            throw err;
         });
 });
 
+router.get('/user/:id(\\d+)/review', (req, res) => {
```
Comment (tim-nguyen-cs) [inline webApp/server_modules/routes/profile.js]: Noel: this can be deleted

File: webApp/views/review.ejs [hunk 2]
```diff
@@ -8,104 +8,126 @@
...
                                         </div>
                                     </div>
-
+                                    
                                     <div class="form-group row category">
-                                        <div class="centralize">
-                                            <button name="submit" type="submit" class="btn btn-primary">Submit Your
-                                                Review</button>
+                                        <div class="centralize" id="submitBut">
+                                            <button class="btn btn-secondary h2-title">Please fill all the fields in this form</button>
```
Comment (tim-nguyen-cs) [inline webApp/views/review.ejs]: MIght want to change this button name

File: webApp/views/review.ejs [hunk 3]
```diff
@@ -8,104 +8,126 @@
...
                     <div class="card">
                         <div class="card-header">
-                            <h2 class="display-4 mb-0 text-center h2-title">Rate Your Tutor</h2>
+                            <h2 class="display-4 mb-0 rating-title">Rate Your Tutor</h2>
                         </div>
                         <div class="card-body">
                             <div class="col-md-auto">
                                 <div>
-                                    <p>You are rating <%= profile.name.first %> <%= profile.name.last %> <%= className %>.</p>
+                                    <p>You are rating <%= profile.name.first %> <%= profile.name.last %> for their teaching of <%= className %>.</p>
```
Comment (tim-nguyen-cs) [inline webApp/views/review.ejs]: I'd rather just tell the user to make sure all fields are filled out here.

**Review by noll115 — APPROVED** [review]
Looks great!

**Pull request merged**